airbyte-integrations/connectors/source-shopify/integration_tests/configured_catalog.json

```diff
@@ -2515,8 +2515,7 @@
                     "type": ["null", "integer"]
                   },
                   "price": {
-                    "type": ["null", "number"],
-                    "multipleOf": 1e-10
+                    "type": ["null", "string"]
```

Comment:
it looks like product can be an object? https://shopify.dev/api/admin/rest/reference/products/product#show-2021-07

Reply:
I'll double check this, thanks.

Reply:
In `product[variants][price]` it's clearly defined as `string` type of output, not object.
There is another `"presentment_prices"`, which is an array of objects with it's own `price` property, so that is the object indeed.

Reply:
Sorry about the confusion then!